.dockerignore

```diff
@@ -0,0 +1,4 @@
+Dockerfile
```

Comment:
I don't think it makes sense. The `Dockerfile` should be versioned with git as well.

Reply:
Dockerfile is not usefull into the image during the build and execution process, this is why it is ignored. I think the root **.git/** folder should also be ignored because also not used during these steps. But maybe it's a good idea to leave it into the docker image because @dstillman plans to implement an auto upgrade feature (but not sure the root source code will be updatable)

Reply:
If we just ignore `.git/` then the `git pull` to update the translators in the image won't work, but we can do a variation of what we do in the <a href="https://github.com/zotero/translation-server/blob/master/Dockerfile">v1 Dockerfile</a>: exclude the translators directory (here in `.dockerignore`), exclude `.git/`, and then `git clone` the translators repo. Then the entrypoint script can do a `git pull` in the translators directory on every startup and it'll always have the latest translators.

Reply:
It's done. I choosed to clone translators at container startup. What do you think?

Reply:
No, there’s no need to pull the whole huge translators repo on each startup. The cloning should happen in the Dockerfile, and it should be updated at container startup.

Reply:
Translators repo is cloning at **fist** startup.

Reply:
I moved the translators repo into the docker image.